Content.Server/Explosion/EntitySystems/ExplosionSystem.Processing.cs

```diff
@@ -474,6 +474,14 @@ private void ProcessEntity(
             }
         }
 
+        // heat the atmosphere
+        float? temperature = fireStacksOnIgnite * 500;
```

Comment:
None of the explosive types other than firebombs should actually have FireStacks in the prototype. 
I think you should just expose a constant temperature hotspot.